packages/front-end/components/Radix/RadioGroup.tsx

```diff
@@ -0,0 +1,89 @@
+import { ReactElement } from "react";
+import { Flex, Text, RadioGroup as RadixRadioGroup } from "@radix-ui/themes";
+import { MarginProps } from "@radix-ui/themes/dist/cjs/props/margin.props";
+import HelperText, { getRadixColor } from "@/components/Radix/HelperText";
+
+export type RadioOptions = {
+  value: string;
+  label?: string | ReactElement;
+  description?: string | ReactElement;
```

Comment:
I've been trying to avoid using `ReactElement` since I think it encourages bad practices like including tooltips and extra text formatting that goes against the design system.

Reply:
Ok, I agree with this. I was just adding this to eventually accommodate what Romain did in the Power Calculator modal, but I'll remove it here.